docs/howto/connection/salesforce.rst

```diff
@@ -0,0 +1,73 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/connection:SalesforceHook:
+
+Salesforce Connection
+=====================
+The HTTP connection type provides connection to Salesforce.
+
+Configuring the Connection
+==========================
```

Comment:
The page should only have one title. Currently, due to bad headings, there are several of them.

Reply:
@mik-laj @feluelle fixed.
Should be OK now